torch/_dynamo/variables/builder.py

```diff
@@ -120,6 +122,14 @@ def load(self, tx):
     def get_examples(self):
         return [self.example]
 
+    def get_fake_examples(self):
+        assert config.fake_tensor_propagation
+        if self.fake_tensor is not None:
+            assert isinstance(
+                self.fake_tensor, torch._subclasses.fake_tensor.FakeTensor
+            )
+        return [self.fake_tensor]
```

Comment:
This is not an example, right? The fake tensor is *canonical* for all possible tensors which would pass the guards, because its sizes/strides have been symintified.

Reply:
Yes